### DISCUSSION TITLE
I have an error

### DISCUSSION THREAD

**Discussion by Igor636965736c610a** [notmarek/BeFake#84] [Q&A]
![image](https://user-images.githubusercontent.com/102369546/234761084-f156a17b-f3a4-4b5e-9a5c-171e8dd62d2a.png)

What can be wrong? it was working fine for me before the update

**Comment by valerierx** [comment]
Are you using the latest revision? Make sure you're on the latest commit by running `pip install git+https://github.com/notmarek/BeFake`

**Reply by Igor636965736c610a** [reply]
Yes, I gave a screen error after the commit you gave 2 hours ago on Issue "can't login"
